src/applications/simple-forms/40-0247/pages/veteranSupportDocs.js

```diff
@@ -4,32 +4,21 @@ import environment from 'platform/utilities/environment';
 
 import { FileField } from 'platform/forms-system/src/js/fields/FileField';
 
+import {
+  createPayload,
+  parseResponse,
+  supportingDocsDescription,
+} from '../helpers';
+
 /** @type {PageSchema} */
 export default {
   uiSchema: {
-    'ui:description': (
-      <>
-        <p>
-          We don’t require that you submit anything with this form. But to speed
-          up the process, we encourage you to submit military records or
-          discharge documents if they’re available.
-        </p>
-        <p>
-          To be eligible for a Presidential Memorial Certificate, the deceased
-          Veteran or Reservist must meet eligibility requirements for burial in
-          a VA national cemetery.
-        </p>
-        <p>Not sure if the Veteran or Reservist is eligible?</p>
-        <p>
-          <a href="/burials-memorials/eligibility/">
-            Check eligibility requirements for burial in a VA national cemetary
-          </a>
-        </p>
-        <p className="vads-u-margin-bottom--4">
-          We prefer that you upload the Veteran’s or Reservist’s DD214.
-        </p>
-      </>
+    'ui:title': (
+      <h3 className="vads-u-margin-y--0">
+        Upload documents (preferably DD214)
+      </h3>
```

Comment:
`titleUI` here? https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/platform/forms-system/src/js/web-component-patterns/titlePattern.js#L23

Reply:
Refactored.  Thanks for catching this!